blackjack_app/src/Component/Menu.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState, useEffect } from 'react'
+import axios from "axios"
+import DrawCard from "./DrawCard"
+
+
+const Menu =()=> {
+
+    const [deckID, setDeckId]=useState(null)
+    const insertID =(e)=>{
+        e.preventDefault()
+        setDeckId(e.target.elements[0].value)
+    }
+
+    const generateID = async()=>{
+        try {
+            let res  = await axios.get("https://deckofcardsapi.com/api/deck/new/shuffle/?deck_count=6")
+            setDeckId(res.data.deck_id)
+        } catch (error) {
+            alert(error)
+        }
+    }
+
+    const display=()=>{
```

Comment:
Good way to error handle. You can also just have an if statement that returns two different things whenever you have a deckId or not, so you don't need to wrap it in a function